invokeai/frontend/web/src/features/nodes/components/NodeGraphOverlay.tsx

```diff
@@ -13,14 +13,16 @@ const NodeGraphOverlay = () => {
       as="pre"
       fontFamily="monospace"
       position="absolute"
-      top={10}
+      top={2}
       right={2}
       opacity={0.7}
       background="base.800"
       p={2}
       maxHeight={500}
+      maxWidth={500}
       overflowY="scroll"
       borderRadius="md"
+      boxShadow="0px 10px 10px rgba(0, 0, 0, 0.25)"
```

Comment:
`boxShadow="0px 10px 10px rgba(0, 0, 0, 0.25)"`

You want to use the colors from the existing theme palette so it translates correctly to all themes. But I need to check if we have it setup for transparency. If not, we check if there's a `shadowOpacity` parameter in chakra to induce it.

Reply:
I couldn't find the one for shadows, so I will just remove the shadow

Reply:
@mickr777 for future reference, [chakra has some built-in shadows](https://chakra-ui.com/docs/styled-system/style-props#shadow)

if you want to add a shadow, you can do it in `invokeai/frontend/web/src/theme/theme.ts`

one thing to note is you cannot use the theme tokens in the shadow string; you have to use CSS variables, as demonstrated in `theme.ts`

then to use the shadow, you'd do `boxShadow='dark.accent'` or `boxShadow='light.accentHover'`.

we aren't using any of these custom shadows anywhere; i usually use the built-in `dark-lg` shadow

Reply:
@psychedelicious thank you for the info